local/test-binary

```diff
@@ -175,4 +175,94 @@ if [[ "${TOKUDB_ENGINES_MTR}" = 'no' ]] && [[ "${HOTBACKUP_TESTING}" = 'no' ]] &
     process_mtr_output "Percona Server - Test" "${WORKDIR_ABS}/small.output" "${WORKDIR_ABS}/small.xml"
 fi
 
+if [[ "${KEYRING_VAULT_MTR}" == 'yes' ]]; then
+    if [ -f /usr/bin/yum ]; then
+        sudo mkdir /run/supervisor
+        supervisorconfigpath=/etc/supervisord.d/vault.ini
+    else
+        supervisorconfigpath=/etc/supervisor/conf.d/vault.conf
+    fi
+    sudo sed -i "s@devtoken1@${VAULT_V1_DEV_TOKEN}@g; s@devtoken2@${VAULT_V2_DEV_TOKEN}@g" $supervisorconfigpath
+    sudo supervisord
+
+    # Required to initialize everything
+    sleep 5
+
+    if [[ ! -z $(supervisorctl status | grep FAILED) ]]; then
+        echo Some process have failed:
+        supervisorctl status
+        exit 1
+    fi
+
+    VAULT_V1_PROD_KEYS="$(/usr/local/bin/vault.0.9.6 operator init -address='http://127.0.0.1:9300')"
```

Comment:
Similarly to the previous PR, try to avoid direct version numbers, use something like `v1` / `v2` instead

Reply:
Fixed in 4e6b1c26dfbc1cc464beffb4e765ddb889500afe